CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## main / unreleased
 
+* [FEATURE] Publish boringcrypto image for amd64. #71
```

Comment:
```suggestion
* [FEATURE] Publish an additional boringcrypto image for linux/amd64,linux/arm64. #71
```
Adding the word "additional" to clarify that this doesn't affect the other image.